### DISCUSSION TITLE
ScreenToGif 2.41

### DISCUSSION THREAD

**Discussion by NickeManarin** [NickeManarin/ScreenToGif#1294] [Announcements]
# What's new?

• Added option to highlight extra mouse buttons clicks.  
• You can now prevent download of updates in metered network connections (@GopherTheCoder).  
• Updated KGySoft libraries, with performance improvements and a new option to execute color operations in linear space (@koszeggy).  

### Fixed:

♦ Fixed issue with Alt + Left/Right not triggering the delete before/after command.   
♦ Fixed issue with FFmpeg encoding sometimes surpassing the 100% mark (@pawlos).      

### More info:

[Known issues](https://github.com/NickeManarin/ScreenToGif/wiki/Help-▪-Known-issues)

PS: The development of successor to ScreenToGif 2 is going strong. 
It's around 75% complete (of its initial set of features), so I expect to be able to launch this year if everything goes right. 
I can't wait to show all the new things alongside its new home page!   

<hr /><em>This discussion was created from the release <a href='https://github.com/NickeManarin/ScreenToGif/releases/tag/2.41'>ScreenToGif 2.41</a>.</em>

**Comment by boromyr** [comment]
> PS: The development of successor to ScreenToGif 2 is going strong. It's around 75% complete (of its initial set of features), so I expect to be able to launch this year if everything goes right. I can't wait to show all the new things alongside its new home page!
Are there any updates on this?

**Reply by NickeManarin** [reply]
Yes, here's the new app:

https://nicke.tech/n-studio
